packages/story-editor/src/app/media/utils/useMediaUploadQueue/test/useMediaUploadQueue.js

```diff
@@ -44,18 +44,18 @@ const mockResource = {
   guid: {
     rendered: 'guid-123',
   },
-  media_details: {
+  mediaDetails: {
     width: 1080,
     height: 720,
   },
-  source_url: 'http://localhost:9876/__static__/asteroid.ogg',
+  sourceUrl: 'http://localhost:9876/__static__/asteroid.ogg',
   title: {
     raw: 'Title',
   },
   description: {
     raw: 'Description',
   },
-  featured_media_src: {},
+  featuredMediaSrc: {},
   meta: {
```

Comment:
No camel case for `meta` keys and `web_stories_is_muted` below?

Reply:
Thanks, Done in https://github.com/google/web-stories-wp/pull/9683/commits/01becc912951a77dc943238c888682ddf3d347fe